README.rst

```diff
@@ -107,19 +110,15 @@ The following creates and activates a new environment named ``diffpy.morph_env``
         conda create -n diffpy.morph_env diffpy.morph
         conda activate diffpy.morph_env
 
-To confirm that the installation was successful, type::
+To confirm that the installation was successful, type ::
 
         python -c "import diffpy.morph; print(diffpy.morph.__version__)"
 
-        The output should print the latest version displayed on the badges above.
+The output should print the latest version displayed on the badges above.
 
 If the above does not work, you can use ``pip`` to download and install the latest release from
 `Python Package Index <https://pypi.python.org>`_.
-To install using ``pip`` into your ``diffpy.morph_env`` environment, we will also have to install dependencies ::
```

Comment:
I think we shouldn't remove this block.  In general, we don't want to remove things coming from the old package, unless it is something that is modified/improved in the new package.